src/oop/programs_with_multiple_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-programs_with_multiple_classes-medium-q1
+
+You want to write a program that models writing tools and stationery objects. Make a Java program with the following requirements:
+
+
+## Pencil
+
+Your `Pencil` class should have the following items:
+
+- Attributes:
+    - An `int` called **hardness**
+    - A `boolean` called **isForSchool**
+
+- Constructors:
+    - A `public Pencil(int, boolean)` constructor to set the
+      **hardness** and **isForSchool** in that order
+
+## Eraser
+
+Your ``Eraser`` class should have the following items:
+
+- Attributes:
+    - An `double` called **size**
+
+- Constructors:
+    - A `public Eraser(double)` constructor to set the
+    - **size** field
+
+## TestPencilCase
+
+- Your `TestPencilCase` class will have a `main` method which initializes two Pencil and a two Eraser, and prints out the
+  attributes of each object. Use the `toString` method provided in each class to do this.
+- The sample output is as follows:
```

Comment:
Same thing with being specific that the values in the sample output are what you need to initialize the class with.